test/ao/sparsity/test_kernels.py

```diff
@@ -10,10 +10,9 @@
 import torch
 import torch.quantization as tq
 
-# Hopefully, the `mo` namespace will move under `nn`
 from torch import nn
 from torch.ao.nn.sparse import quantized as ao_nn_sq
-from torch.ao.nn.sparse.quantized.utils import QNNPACKLinearBlockSparsePattern
+from torch.ao.nn.sparse.quantized.utils import LinearBlockSparsePattern
```

Comment:
Please add a TODO: In a later PR where we modify from_float to take the sparsity pattern as an argument and this pattern will not be needed.